dockers/base-cuda/Dockerfile

```diff
@@ -102,7 +102,7 @@ RUN \
     # install ColossalAI
     # TODO: 1.13 wheels are not released, remove skip once they are
     if [[ $PYTORCH_VERSION != "1.13" ]]; then \
-        pip install "colossalai==0.2.3"; \
+        pip install "colossalai==0.2.4"; \
```

Comment:
0.2.4 is the one i tested yesterday in #16747. They are releasing daily so let's wait a bit. This would have to be updated in our req files as well. This fix here is for @justusschock who discovered an error.